go/kbfs/libkbfs/node_cache.go

```diff
@@ -444,3 +442,16 @@ func (ncs *nodeCacheStandard) AddRootWrapper(f func(Node) Node) {
 	defer ncs.lock.Unlock()
 	ncs.rootWrappers = append(ncs.rootWrappers, f)
 }
+
+func (ncs *nodeCacheStandard) SetObfuscatorMaker(
```

Comment:
Why did you pull this out of the constructor?

Reply:
Because the `folderBranchOps` constructor makes a `nodeCache` first before constructing itself, so passing in an `fbo` member function to the `nodeCache` constructor isn't possible.